webapi/payfee.go

```diff
@@ -164,8 +164,8 @@ findAddress:
 		return
 	}
 
-	wifAddr, err := dcrutil.NewAddressPubKeyHash(dcrutil.Hash160(votingWIF.PubKey()), cfg.NetParams,
-		dcrec.STEcdsaSecp256k1)
+	pkHash := dcrutil.Hash160(votingWIF.PubKey())
```

Comment:
I'd suggest using `stdaddr.Hash160` here now.  This is fine for now, but the `dcrutil` version will likely eventually go away and since it's for the purposes of an address, it seems to make more sense to use `stdaddr` for it.